scripts/development/m3_stack/start_m3.sh

```diff
@@ -188,11 +193,6 @@ if [[ "$AGGREGATOR_PIPELINE" = true ]]; then
     echo "Done validating topology"
 
     # Do this after placement for m3coordinator is created.
```

Comment:
Not sure if I follow, I moved this init command because agg instance will die if topic is not created within wait time. So I'm going to init the topic before starting agg